gitgud/test_operator.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from gitgud.skills import all_skills
+from gitgud.operations import get_operator
+
+
+@pytest.fixture
+def file_operator(gg):
+    file_operator = get_operator()
+    assert file_operator is not None
+    return file_operator
+
+
+@pytest.fixture(scope='function')
+def progress_data(file_operator, gg):
+    progress_data = file_operator.read_progress_file()
+    return progress_data
+
+
+def test_get_level_progress(file_operator, progress_data):
+    level = all_skills["1"]["1"]
+    assert file_operator.get_level_progress(level) in {"complete", "incomplete", "unvisited"}  # noqa: E501
```

Comment:
Can you split this to multiple lines? In general, I prefer not adding E501 exceptions except for user messages